concepts/control-flow/.meta/config.json

```
@@ -1,7 +1,5 @@
 {
-    "blurb": "<todo>",
-    "authors": [
-        "0xNeshi"
-    ],
-    "contributors": []
+  "blurb": "Control flow in Cairo uses if expressions to conditionally execute code and loops to repeat code while a condition is true.",
+  "authors": ["misicnenad", "Falilah"],
+  "contributors": []
```

```suggestion
  "contributors": ["0xNeshi"]
```
I changed my GH username